tools/idr_download/idr_download_by_ids.xml

```diff
@@ -33,7 +33,7 @@
         | python '$__tool_directory__/idr_download_by_ids.py' 
         #set $channel = str($channel).strip()
         #if $channel:
-            -c $channel
+            -c '$channel'
```

Comment:
I discovered some channels have strange characters, so quotes are needed.

Reply:
When you're saying strange names, do we also need additional chars allowed in the text box then?

Reply:
That's a good point, I didn't investigate this. I just happened to stumble across an example with `()`.

Reply:
I don't really know how to get a list of all channel names to work out if this is necessary. I guess we can assume it's not and update the tool later if we need to.